### PR TITLE
Update Reformation scrape script to grab product URLs

### PR DESCRIPTION
**Pull request by sanjayesn**
This PR allows us to scrape product URLs for each Reformation top, so we can include the URL in product recommendations. I did not allow the script to run to completion so as to not overwhelm our database with duplicates. Here are some examples of results to show we can scrape product URLs as intended:

https://console.firebase.google.com/project/suzie-1234/firestore/data~2Fsustainable_alternatives~2Fbrands~2Freformation~2FhadFVgtiNnJyf7kv9Cir

https://console.firebase.google.com/project/suzie-1234/firestore/data~2Fsustainable_alternatives~2Fbrands~2Freformation~2FKID49OGPMzLWMP069iMD

Moving forward, we should come up with a plan to eliminate duplicate entries in Firestore and standardize our data flow.

### CODE REVIEW

File: Bulk Scraping/ReformationScrape.py [hunk 1]
```diff
@@ -7,31 +7,38 @@
 
 # When running this you'll have to download the service account info, not sure 
 # how to do this well with version control. 
-cred = credentials.Certificate('/Users/trey/Documents/suzie-1234-firebase-adminsdk-osp8h-1e8460abbb.json')
+cred = credentials.Certificate('suzie.json')
```
Comment (sanjayesn) [inline Bulk Scraping/ReformationScrape.py:10]: You need to use a local copy of the JSON credentials file which you can get via the Firebase console.

File: Bulk Scraping/ReformationScrape.py [hunk 2]
```diff
@@ -7,31 +7,38 @@
...
-    print('PAGE', page_no)
-    results = soup.find_all("div", "product-summary")
-    for item in results:
-        #Navigating down to the image source
-        image_url = item.div.a.img.attrs['data-product-summary-hover-image']
-        result_dict = item.attrs
+    for product_summary, media_link in zip(product_summary_results, media_link_results):
+        # Navigating down to the image source
+        image_url = product_summary.div.a.img.attrs['data-product-summary-hover-image']
+        result_dict = product_summary.attrs
```
Comment (treyholt) [inline Bulk Scraping/ReformationScrape.py:30]: Is this updated with any slight tweaks we made when we ran this the other day?

**Review by treyholt — COMMENTED** [review]
Quick Q, otherwise lgtm

File: Bulk Scraping/ReformationScrape.py (hunk 2 above)
Comment (sanjayesn) [reply Bulk Scraping/ReformationScrape.py:30]: Will add them right now!

**Review by treyholt — APPROVED** [review]
looks good!

**Pull request merged**